pkg/networkservice/chains/xconnectns/server.go

```diff
@@ -69,13 +70,14 @@ func NewServer(ctx context.Context, name string, authzServer networkservice.Netw
 		stats.NewServer(ctx),
 		// Statically set the url we use to the unix file socket for the NSMgr
 		clienturl.NewServer(clientURL),
+		heal.NewServer(ctx, addressof.NetworkServiceClient(adapters.NewServerToClient(rv))),
 		connect.NewServer(
 			ctx,
 			func(ctx context.Context, cc grpc.ClientConnInterface) networkservice.NetworkServiceClient {
 				return client.NewClient(ctx,
 					cc,
 					client.WithName(name),
-					client.WithHeal(addressof.NetworkServiceClient(adapters.NewServerToClient(rv))),
+					heal.NewClient(ctx, cc),
```

Comment:
Isn't heal built into client.NewClient?  If not... why not?

Reply:
Yep, it is already there. My mistake

Reply:
Please do check further downstream as well.  When I attempted this fix in my local repo, the cmd-forwarder-vpp stopped working... so we may need to chase down issues as this trickles downstream (although I could have also just gotten my local repo into an odd state).